EFWorkshop/EFWorkshopCodeFirst/DataAccess/UniversityDbContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using EfWorkshop.DataAccess.Models;
+
+namespace EfWorkshop.DataAccess
+{
+    internal class UniversityDbContext : DbContext
+    {
+        private const string ConnectionString = "Server=localhost;Database=University;Trusted_Connection=True;TrustServerCertificate=True;";
+        public DbSet<Faculty> Faculties { get; set; }
```

Comment:
Please add empty lines here to separate the private field from auto properties and the properties from the constructor.